File: src/ed/ogonzalezm/a06/NodoDL.java
/*
 * To change this license header, choose License Headers in Project Properties.
 * To change this template file, choose Tools | Templates
 * and open the template in the editor.
 */
package ed.ogonzalezm.a06;

/**
 *
 * @author HONORIO ZAIBACK
 */
public class NodoDL <T>{
    T data;
    NodoDL<T> next;
    NodoDL<T> back;
    
    NodoDL (T d) { 
        this.data = d;
        this.next = this.back = null;
    } 
}

File: src/ed/ogonzalezm/a06/NodoDL.cpp
#include "NodoDL.h"
#include <string>

template <typename T> class NodoDL {
public:
	T data;
	NodoDL* next;
	NodoDL* back;

	NodoDL(T d) {
		data = d;
		next = nullptr;
		back = nullptr;
	}
};
